internal/api_client/cloud_builder_client.go

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright 2023 VMware, Inc.
+ *    SPDX-License-Identifier: MPL-2.0
+ */
+
+package api_client
+
+import (
+	"crypto/tls"
+	openapiclient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	vcfclient "github.com/vmware/vcf-sdk-go/client"
+	"net/http"
+)
+
+// CloudBuilderClient is an API client that can execute the APIs of the CloudBuilder appliance.
+// Supports only HTTP Basic Authentication.
+type CloudBuilderClient struct {
+	username           string
+	password           string
+	cloudBuilderUrl    string
+	ApiClient          *vcfclient.VcfClient
+	allowUnverifiedTls bool
+}
+
+func NewCloudBuilderClient(username, password, url string, allowUnverifiedTls bool) *CloudBuilderClient {
+	result := &CloudBuilderClient{
+		username:           username,
+		password:           password,
+		cloudBuilderUrl:    url,
+		allowUnverifiedTls: allowUnverifiedTls,
+	}
+	result.init()
+	return result
+}
+
+func (cloudBuilderClient *CloudBuilderClient) init() {
+	// Disable cert checks
```

Comment:
Why this comment "Disable cert checks"? I thought you are disabling only when allowUnverifiedTls=true

Reply:
Wouldn't a Cloud Builder instance always be unverified? Or do we support changing it's certificate, even if it is ephemeral.

Reply:
If Cloud Builder instance is always unverified, then I'll ignore the allowUnverifiedTls provider configuration and pass "true" always.